modules/ROOT/Edits/Configuring and Managing Message Flows/undeploy-message-flows.adoc

```diff
@@ -0,0 +1,23 @@
+== Undeploying Message Flows
+
+Undeploying a message flow prevents future messages from being processed and enables you to recover resources that are consumed by a message flow that you no longer need.
+
+You can undeploy your message flow from either a sandbox or a production environment. The undeployment process takes about five minutes, during which actions such as `Deploy` and `Delete` are unavailable.
+
+[IMPORTANT]
+During the undeployment process, do not log out of Anypoint Partner Manager or close your browser window.
+
+Before you undeploy message flows, ensure that your on-premises Mule instances, the servers hosting them, and the server group in Anypoint Runtime Manager called `B2BserverGroup` are active and running.
+
+To undeploy a message flow:
+
+. In the navigation menu, select *Message Flows*.
+. Select the message flow to undeploy from the list and click *Undeploy*.
+
+Partner Manager checks to see if any of the applications deployed to Mule are orphaned as a result of the undeployment. It cleans up orphaned applications to release any resources the applications were consuming.
```

Comment:
Consider omitting the possibly unclear "It" reference by making this one sentence or by renaming Partner Manager.